connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointTask.java

```diff
@@ -395,7 +401,7 @@ void syncGroupOffset(String consumerGroupId, Map<TopicPartition, OffsetAndMetada
     Map<String, Map<TopicPartition, OffsetAndMetadata>> getConvertedUpstreamOffset() {
         Map<String, Map<TopicPartition, OffsetAndMetadata>> result = new HashMap<>();
 
-        for (Entry<String, Map<TopicPartition, Checkpoint>> entry : checkpointsPerConsumerGroup.entrySet()) {
+        for (Entry<String, Map<TopicPartition, Checkpoint>> entry : checkpointsStore.contents().entrySet()) {
             String consumerId = entry.getKey();
```

Comment:
this getConvertedUpstreamOffset could be moved to CheckpointStore since it only depends on the checkpoint store state.